lib/src/features/settings/metrics_service.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:fit_and_healthy/shared/models/WeightEntry.dart';
+import 'package:fit_and_healthy/shared/models/activity_level.dart';
+import 'package:fit_and_healthy/shared/models/gender.dart';
+import 'package:fit_and_healthy/shared/models/weight_goal.dart';
+
+class MetricsService {
```

Comment:
A service class should only have the responsibility of reading and writing to persistent storage. Most often used as a intermediary between the controller and the repositories when you need to aggregate data from several repositoreis. Depending on our DB structure you could probably rename this to MetricsController.

You can look at SettingsController and SettingsService to see the start of an example